src/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  left: 5px;
+  margin-top: 15px;
+  width: 100%;
+  top: 3px;
+}
+
+.search-form__input {
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  padding-left: 15px;
```

Comment:
Why do you set it to `15px`? It's not the correct value for any of the inputs. Plus, you just rewrite it in other selectors.
```suggestion
```